### PR TITLE
added navigation by arrows inside context menu

### PR DESCRIPTION
**Pull request by Yura13**
added navigation by arrows inside context menu

### CODE REVIEW

File: src/lib/contextMenuContent.component.ts [hunk 1]
```diff
@@ -216,6 +226,62 @@ export class ContextMenuContentComponent implements OnInit, OnDestroy, AfterView
     this.changeDetector.markForCheck();
   }
 
+  @HostListener('document:keydown', ['$event'])
```
Comment (isaacplmann) [inline src/lib/contextMenuContent.component.ts]: You can let angular handle a lot of this KeyboardEvent logic for you by using listeners like `document:keydown.ArrowDown` on the relevant functions directly.

**Comment by isaacplmann** [comment]
Good work solving the tabbing issue by tracking the `activeMenuItemIndex` by the way.  That solves a big piece of this.

**Comment by Yura13** [comment]
Could you help with your comments?
I added you as a contributor of my forked repository.

**Comment by isaacplmann** [comment]
I'm merging this in to finish working on it.  Thanks for your help!

**Pull request merged**